.kitchen.yml

```diff
@@ -31,6 +31,15 @@ platforms:
   - name: windows-2012r2
     driver_config:
       box: chef/windows-server-2012r2-standard
+# Solaris 10 requires the OS ISO to be mounted into `/cdrom` in order for the
```

Comment:
Is there some magical solaris way to mount an iso from within the solaris box? It might be worth shipping the ISO in the atlas box and having it automount if that's possible